virtualmachine/vsphere/constants.go

```diff
@@ -0,0 +1,64 @@
+package vsphere
+
+const STATICIP_CUSTOM_SPEC_NAME = "static-ip-libretto"
+const XML_STATIC_IP_SPEC = `
+<ConfigRoot>
+  <_type>vim.CustomizationSpecItem</_type>
+  <info>
+    <_type>vim.CustomizationSpecInfo</_type>
+    <changeVersion>1505235815</changeVersion>
+    <description/>
+    <lastUpdateTime>2017-09-12T17:03:35Z</lastUpdateTime>
+    <name>static-ip-libretto</name>
+    <type>Linux</type>
+  </info>
+  <spec>
+    <_type>vim.vm.customization.Specification</_type>
+    <globalIPSettings>
+      <_type>vim.vm.customization.GlobalIPSettings</_type>
+      <dnsServerList>
+        <_length>1</_length>
+        <_type>string[]</_type>
+        <e id="0">8.8.8.8</e>
+      </dnsServerList>
+      <dnsSuffixList>
+        <_length>1</_length>
+        <_type>string[]</_type>
+        <e id="0">google.com</e>
+      </dnsSuffixList>
+    </globalIPSettings>
+    <identity>
+      <_type>vim.vm.customization.LinuxPrep</_type>
+      <domain>gsintlab.com</domain>
```

Comment:
Does all these hardcoded xml values gets updated correctly (for an entirely different vcenter setup and configuration) ?

Reply:
Yes, tried it on dev-vcenter and starling-vcenter. The custom spec is created and vm is assigned to the vm.